contracts/DelFiPrice.sol

```diff
@@ -3,23 +3,32 @@ pragma experimental ABIEncoderV2;
 
 import "./OpenOraclePriceData.sol";
 import "./OpenOracleView.sol";
+import "./SafeMath.sol";
 
 /**
  * @notice The DelFi Price Feed View
  * @author Compound Labs, Inc.
  */
 contract DelFiPrice is OpenOracleView {
+    using SafeMath for *;
```

Comment:
might as well using "SafeMath for uint256" here since it will revert for everything else. Interesting that this is an option though

Reply:
oooh it upcasts things? interesting.